docs(changelog): add missing changelog entry (#433)
bytebeamio/rumqtt: CHANGELOG.md

---
rumqttc
---

> - BREAKING: Changed the way we re-export `ClientConfig` struct (#428)

Author: not sure actually, wdyt?

> - BREAKING: Changed the way we re-export `ClientConfig` struct (#428)

Reviewer: ```suggestion
- Ensure re-export of `ClientConfig` is clearly from the `tokio_rustls` crate (#428)
```
